internal/objects/service/service.go

```diff
@@ -262,10 +278,28 @@ func DesiredEnvoyService(contour *operatorv1alpha1.Contour) *corev1.Service {
 		svc.Spec.Type = corev1.ServiceTypeLoadBalancer
 		provider := contour.Spec.NetworkPublishing.Envoy.LoadBalancer.ProviderParameters.Type
 		lbAnnotations := LbAnnotations[provider]
+		isInternal := contour.Spec.NetworkPublishing.Envoy.LoadBalancer.Scope == operatorv1alpha1.InternalLoadBalancer
+		delete(lbAnnotations, awsLbEipAnnotation)
```

Comment:
`delete` statements added, since otherwise operator carries those annotations with it once `Contour` custom resource is updated or recreated.